cmake/onnxruntime_providers.cmake

```diff
@@ -430,9 +430,9 @@ if (onnxruntime_USE_DML)
   add_dependencies(onnxruntime_providers_dml ${onnxruntime_EXTERNAL_DEPENDENCIES})
   target_include_directories(onnxruntime_providers_dml PRIVATE ${ONNXRUNTIME_ROOT} ${ONNXRUNTIME_ROOT}/../cmake/external/wil/include)
   
-  target_link_libraries(onnxruntime_providers_dml ${CMAKE_CURRENT_BINARY_DIR}/packages/DirectML.0.0.1/build/DirectML.targets)
+  target_link_libraries(onnxruntime_providers_dml ${REPO_PACKAGE_LOCATION}/DirectML.0.0.1/build/DirectML.targets)
```

Comment:
chatted with Ori about this change, thought it might be unnecessary now. can you confirm?

Reply:
right.   we rolled this back.

---
In reply to: [354033539](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354033539) [](ancestors = 354033539)